wren-modeling-rs/core/src/mdl/mod.rs

```diff
@@ -211,7 +213,12 @@ pub async fn transform_sql_with_ctx(
     match plan_to_sql(&analyzed) {
         Ok(sql) => {
             // TODO: workaround to remove unnecessary catalog and schema of mdl
-            let replaced = sql.to_string().replace(&catalog_schema, "");
+            let replaced = sql
+                .to_string()
+                .replace(&catalog_schema, "")
+                // TODO: There're some issue about datafusion unparsing the expr column name with different case
+                // The workaround is to normalize the SQL to the lower case to support Wren View.
+                .to_lowercase();
```

Comment:
I'll file the corresponding issue to Datafusion. It generate an invalid SQL for view like
```sql
SELECT 
  Revenue_orders.Order_id, 
  Revenue_orders."sum(Order_items.Price)"  # It's generated from the aggregation in the subquery
FROM 
  (
    SELECT 
      Order_items.Order_id, 
      sum(Order_items.Price) 
    FROM 
      (
        SELECT 
          Order_items.Order_id, 
          Order_items.Price 
        FROM 
          (
            SELECT 
              datafusion.public.order_items.order_id AS Order_id, 
              datafusion.public.order_items.price AS Price 
            FROM 
              datafusion.public.order_items
          ) AS Order_items
      ) AS Order_items 
    GROUP BY 
      Order_items.Order_id
  ) AS Revenue_orders
```